src/components/ui/Tree/Tree.tsx

```
@@ -0,0 +1,34 @@
+        {items.map((item: any) => (
+            <>
+                <TreeItem key={item.label} expanded={item.expanded}>
+                    {item.label}
+                </TreeItem>
+                <>
+                    {item.items && item.expanded && item.items.map((subItem: any) => (
+                        <TreeItem className='ml-4' key={subItem.label} expanded={subItem.expanded}>
+                            {subItem.label}
+                        </TreeItem>
+                    ))}
+                </>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)